_source/logzio_collections/_metrics-sources/Prometheus-cloudwatch-exporter.md

```diff
@@ -6,22 +6,41 @@ logo:
 data-source: Amazon CloudWatch for Prometheus metrics
 templates: ["docker-metricbeat"]
 open-source:
-  - title: 
-    github-repo: 
+  - title: CloudWatch metrics for Prometheus
+    github-repo: logz-aws-metrics
 contributors:
   - yotamloe
   - yberlinger
 shipping-tags:
-  - aws
+  - prometheus
 ---
 
-# logzio-aws-metrics
-
-**IMPORTANT ⚠️ :** This project is intended for specific private beta customers.
+{% include page-info/early-access.md type="beta" %}
 
 This project allows you to collect Prometheus format metrics from Amazon CloudWatch with the CloudWatch exporter, and ship them to Logz.io using the OpenTelemetry collector.
 
+We simplify the data export and collection for your metrics: You tell us the desired namespaces and regions you want to send your data from and we fetch the most relevant metrics to display on the Logz.io pre-built Infrastructure Monitoring dashboards.
+
+
 ### Schema
+
+Within the applications included in the Docker composefile : 
+
+1. Python writes your environment data to configuration files shared with both the CloudWatch exporter and the OpenTelemetry collector containers. 
+    
+    The configuration files specify: 
+    
+    * Which platforms the data is collected from
```

Comment:
I think we can remove this line because it's only for was cloudwatch not any other platform